server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/mailet/filter/JMAPFilteringTest.java

```diff
@@ -1057,4 +1059,79 @@ void actionShouldCombineFlags(JMAPFilteringTestSystem testSystem) throws Excepti
         assertThat(StorageDirective.fromMail(Username.of("recipient1"), mail).getFlags().get())
             .isEqualTo(expectedFlags);
     }
+
+    @Test
+    void shouldMatchRuleOfAndConditionCombiner(JMAPFilteringTestSystem testSystem) throws Exception {
+        Mono.from(testSystem.getFilteringManagement().defineRulesForUser(RECIPIENT_1_USERNAME,
+            Optional.empty(),
+            Rule.builder()
+                .id(Rule.Id.of("1"))
+                .name("rule 1")
+                .conditionGroup(Rule.ConditionGroup.of(Rule.ConditionCombiner.AND, Rule.Condition.of(FROM, CONTAINS, USER_2_USERNAME), Rule.Condition.of(SUBJECT, CONTAINS, "cd")))
+                .action(Rule.Action.of(Rule.Action.AppendInMailboxes.withMailboxIds(testSystem.getRecipient1MailboxId().serialize()),
+                    false, false, false, ImmutableList.of()))
```

Comment:
Use a constant in RuleFixture for the action

Reply:
Could not find the same thing in RuleFixture. Moreover, RuleFixture is in a different module.